typescript/opensearch/cwlogs_ingestion/resources/pipeline/configuration.yaml

```diff
@@ -0,0 +1,15 @@
+version: "2"
+cwlogs-ingestion-http-pipeline:
+  source:
+    http:
+      path: /logs/ingest
+  sink:
```

Comment:
I think this pipeline example would be more helpful if we ran `grok`. See my comment above about application logs. We could possibly have a log that you could grok here.